scripts/build.sh

```diff
@@ -12,7 +12,7 @@
 # in subsurface/build)
 #
 # by default it puts the build folders in
-# ./subsurface/llibdivecomputer/build (libdivecomputer build)
+# ./subsurface/libdivecomputer/build (libdivecomputer build)
```

Comment:
not sure this qualifies as "disturbing" ("causing anxiety; worrying"), but yeah, typo.

Reply:
updated text in PR (it is not present in the commit text).